src/rgw/rgw_admin.cc

```diff
@@ -309,15 +309,18 @@ void usage()
   cout << "  mfa remove                 delete MFA TOTP token\n";
   cout << "  mfa check                  check MFA TOTP token\n";
   cout << "  mfa resync                 re-sync MFA TOTP token\n";
-  cout << "  topic list                 list bucket notifications topics\n";
-  cout << "  topic get                  get a bucket notifications topic\n";
-  cout << "  topic rm                   remove a bucket notifications topic\n";
+  cout << "  topic list                 list notifications topics\n";
```

Comment:
i think the bst term here is "bucket notification topics"
these are topics related to the "bucket notifications"